framer/LayerDraggable.coffee

```diff
@@ -278,6 +278,9 @@ class exports.LayerDraggable extends BaseClass
 		# (which would return a stale value before the simulation had finished one tick)
 		# and because @_start currently calls calculateVelocity().
 		@_isDragging = false
+
+		# reset isMoving if not animating, otherwise animation start/stop will reset it
+		@_isMoving = @_isAnimating
```

Comment:
Why go through the getter if it returns `@_isAnimating || false`? especially when manipulating a similar private property?